lib/core/sync_job_runner.rb

```diff
@@ -25,7 +25,7 @@ def initialize(connector_settings, service_type)
       @connector_settings = connector_settings
       @sink = Core::OutputSink::EsSink.new(connector_settings.index_name)
       @connector_class = Connectors::REGISTRY.connector_class(service_type)
-      @connector_instance = Connectors::REGISTRY.connector(service_type)
+      @connector_instance = Connectors::REGISTRY.connector(service_type, connector_settings)
```

Comment:
It seems for a connector, it only needs `connector_settings.configuration`, instead of `connector_settings`